package.json

```diff
@@ -44,10 +44,13 @@
     "cheerio": "0.22.0",
     "colors": "1.1.2",
     "commander": "2.9.0",
+    "emojilib": "2.2.0",
     "event-stream": "3.3.4",
     "express": "4.15.2",
     "fs-extra": "2.1.2",
     "irc-framework": "2.6.1",
+    "jquery-textcomplete": "1.8.0",
+    "json-loader": "0.5.4",
```

Comment:
`emojilib` can't import without this since in emojilib codebase, there is a require of a json file.